.env.example

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
I think you need to add backend URL as environment variable too

Reply:
added